tests/update/zypper_ref_auto_import_keys.pm

```diff
@@ -0,0 +1,21 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: zypper
+# Summary: auto import gpg keys
+# Maintainer: Shukui Liu <skliu@suse.com>
```

Comment:
Nowadays we use `# Maintainer: QE YaST <qa-sle-yast@suse.de>`, as all the team is responsible for the code. 
I would also complete the summary with more explanation: "Auto import gpg keys, useful when we re-launch the test suite after the gpg key for maintenance repositories have expired (eg next day after initial run)."

Reply:
fixed.